stock_split_picking/models/stock_picking.py

```diff
@@ -78,3 +80,19 @@ def split_process(self):
                     'picking_id': backorder_picking.id,
                 })
                 new_moves._action_assign()
+
+                # Propagate split to chained picking/moves
+                chained_moves = picking.move_lines.mapped('move_dest_ids')
+                for chained_move in chained_moves:
```

Comment:
We probably don't want to propagate the split if `len(chained_moves) > 1`. But I'm not sure yet if we should raise an error or not.

Reply:
I think we can have more than one chained move here and should handle this case. For instance if the picking contains several products (then several moves) => the split might apply on several moves and cause several chained moves.

Reply:
Well, my comment wasn't really clear, but if each move is linked to only `move_dest_ids` we should propagate. If a move is linked to multiple `move_dest_ids`, there's no possibility to tell which move should be split. :wink: 
Therefore you might have to loop `picking.move_lines` instead of `chained_moves` and proceed from there :
```
for move in picking.move_lines:
    if len(move.move_dest_ids) > 1:
        continue
```

Reply:
I will investigate this hint, thanks!